lib/model/unittest/CResourceMonitorTest.cc

```diff
@@ -21,6 +23,22 @@ BOOST_AUTO_TEST_SUITE(CResourceMonitorTest)
 using namespace ml;
 using namespace model;
 
+namespace {
+using TTokenListDataCategorizerKeepsFields =
+    model::CTokenListDataCategorizer<true,  // Warping
```

Comment:
named constants would be better here I think...

Reply:
Actually since in this test it doesn't matter exactly how the categorizer is configured I switched it to use the default template arguments in this file.  Then this whole unnamed namespace goes.